src/main/java/org/ecocean/servlet/IndividualAddEncounter.java

```diff
@@ -306,71 +306,7 @@ public static void executeEmails(Shepherd myShepherd, HttpServletRequest request
     }
 
     private void youTubePostback(Encounter enc2add, Shepherd myShepherd, String context) {
```

Comment:
why is this being maintained? couldn't we remove all of youTubePostback?

Reply:
removed this, as well as stuff in `webapp/WEB-INF/data`.  i _did_ leave these, as they were the only files without `sutime` in their names, but likely they can go too. am i 100% sure? no. but close. so i left them.

```
$ ls -lt src/main/webapp/WEB-INF/data/english-left3words-distsim.tagger*
-rw-rw-r-- 1 jon jon 12409329 Jul 29 16:00 src/main/webapp/WEB-INF/data/english-left3words-distsim.tagger
-rw-rw-r-- 1 jon jon     1522 Jul 29 16:00 src/main/webapp/WEB-INF/data/english-left3words-distsim.tagger.props
```